index.js

```diff
@@ -1,4 +1,4 @@
-const Hashids = require('hashids');
+const Hashids = require('hashids').default;
```

Comment:
Is using the `default` property on the hashids package going to break anything? I can't seem to see a reference to that property in their docs.

Reply:
They're using ESM's `export default` now. When this translates to CJS, you get an object that looks like `{ default: HashIds }`